src/python/py_llm_pipeline.cpp

```diff
@@ -96,7 +96,14 @@ void init_llm_pipeline(py::module_& m) {
             const std::map<std::string, py::object>& config
```

Comment:
should we add `kwargs` as it was in that PR?

Reply:
Currently `LLMPipeline` has 2 `init()`'s with `config` param and 2 separate `init()`s with `kwargs` which were added in this PR: https://github.com/openvinotoolkit/openvino.genai/commit/c11d9a9b60a38744f5ffb615680231c2de3f02d8#diff-1e7da7f7c47c67afe962fff6f2284adafb1cc1e0e997e09d7b26ba5592735c80R429

So it looks redundant to add `kwargs` to `init()`s with `config`. We already can pass `kwargs` to `LLMPipeline` on master. 

So unless we want to support syntax with both `config` param and `kwargs` (example below) `kwargs` are not needed here.

```
enable_compile_cache = {...}
pipe = openvino_genai.VLMPipeline(args.model_dir, device, enable_compile_cache, ENABLE_MMAP=True)
```

Reply:
Just noticed your comment to with suggestion merge them. I am on it.